terraform/groups/lambda/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,11 @@
+# ------------------------------------------------------------------------------
+# Environment
+# ------------------------------------------------------------------------------
+
+environment = "cidev"
+aws_profile = "development-eu-west-2"
+cron_schedule = "cron(0 23 * * ? *)"
+aws_region = "eu-west-2"
+release_bucket_name = "development-eu-west-2.release.ch.gov.uk"
+remote_state_bucket = "ch-development-terraform-state-london"
```

Comment:
We shouldn't be using this bucket

Reply:
This is only for reading the existing remote state of the Test And Development network - this is it's current location.
We're using the container / new scripts for applying changes, so the remote state of this repo will be set there.
👍